util/mkerr.pl

```diff
@@ -455,6 +453,10 @@ sub help
 #ifndef HEADER_${lib}ERR_H
 # define HEADER_${lib}ERR_H
 
+# ifndef HEADER_SYMHACKS_H
```

Comment:
Side note: Not really sure why that's needed...

(no, you don't have to change anything...)